### PR TITLE
Screen program mapping is now opt-in (as it slows down exits from insert back to normal mode)

### PR DESCRIPTION
**Pull request by rekendahl**
To enable screen mapping add `let g:spf13_screen_mapping = 1` in your .vimrc.before

### CODE REVIEW

File: .vimrc [hunk 1]
```diff
@@ -570,7 +574,7 @@
         nnoremap <silent> <leader>gi :Git add -p %<CR>
         nnoremap <silent> <leader>gg :SignifyToggle<CR>
     "}
-
```
Comment (rekendahl) [inline .vimrc:573]: removed some annoying spaces as well to make the file not show syntax issues

**Comment by rekendahl** [comment]
See #510 for discussion of this pull request

File: .vimrc.before [hunk 2]
```diff
@@ -85,6 +85,13 @@
     "   let g:multi_cursor_quit_key='<Esc>'
     " Require a special keypress to enter multiple cursors mode
     "   let g:multi_cursor_start_key='+'
+
+    " Add mappings for Unix `screen` program.
```
Comment (johntyree) [inline .vimrc.before]: I know this comment is already long, but it doesn't say what this option actually does. What are we mapping here? Home and End? Does anyone know?

Comment (rekendahl) [inline .vimrc.before]: I have no idea. I was mostly interested in making them "go away" as they cause an almost second delay when exiting insert mode. 

I could easily see simply removing the whole thing! 

I don't think we should be focusing on external programs like screen and TMUX and instead leave that to the user.

**Comment by adambiggs** [comment]
+1 for removing it entirely.

**Comment by rekendahl** [comment]
Ok. Let me do a pull request removing it entirely. I vote for that too. We need to minimize what we have (spf-vim is on the upper end of content) and it seems like reducing assumption on external flows makes sense

**Comment by rekendahl** [comment]
Ok. Updated pull request: https://github.com/rekendahl/vimeda/commit/9b2363616e46ee59ac4c0c0b513c989864471bef

Simply removes these mappings. If anyone is using them I hope the chime in

**Comment by johntyree** [comment]
I followed the history back to where this was introduced. @taxilian if you're still using this, be aware that it's gone now.

**Pull request merged**